### PR TITLE
test: Add riscv64 tests

### LINKED ISSUE #14: Add RISC-V 64 test
@Skallwar maybe?

See https://github.com/andre-richter/qemu-exit/commit/33828d5f8eacd95c2a1a600eb9ee3545539eeee8

The test workflow currently has no matrix because I did not know how to pair two values together.
You'd need target+linker file.

### PR DESCRIPTION
**Pull request by Skallwar**
Closes #14

### CODE REVIEW

**Comment by Skallwar** [comment]
Yes! This is working @andre-richter

File: tests/riscv64_virt/link.ld [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+ENTRY(_start)
+
+SECTIONS
+{
+    /* Set current address to the address where OpenSBI will jump */
+    . = 0x80200000;
+
+    .text : {
+        *(.text._start)
```
Comment (andre-richter) [inline tests/riscv64_virt/link.ld:14]: test._start should be the only one needed. All the other can be deleted.

**Review by andre-richter — COMMENTED** [review]
Where is the `fw_jump.elf ` from? Are we allowed to redistribute it? Do we need to mention some license in the README or something?

File: tests/riscv64_virt/link.ld (hunk 1 above)
Comment (Skallwar) [reply tests/riscv64_virt/link.ld:14]: Will try

**Comment by Skallwar** [comment]
> Where is the fw_jump.elf from? 

[OpenSBI](https://github.com/riscv/opensbi).

OpenSBI is under [BSD 2 license](https://chttps://choosealicense.com/licenses/bsd-2-clause/hoosealicense.com/licenses/bsd-2-clause/) so:
> Are we allowed to redistribute it?

Yes
> Do we need to mention some license in the README or something?

Yes, I will add a mention in the readme and the BSD2 license file in the directory of fw_jump.elf

File: tests/riscv64_virt/link.ld (hunk 1 above)
Comment (Skallwar) [reply tests/riscv64_virt/link.ld:14]: .rodata is necessary otherwise there is .rodata in the first LOAD phdr and ``_start`` is not at 0x80200000

**Comment by andre-richter** [comment]
Thanks. This is an awesome crate now, I'll release this as 1.0.0.

**Pull request merged**